src/formatDate.js

```diff
@@ -8,7 +8,25 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateComponents = date.split(fromFormat[3]);
```

Comment:
It's good that you're using the separator from the `fromFormat` to split the date. This makes the function flexible for different separators.